triton-client/src/store/thunks.ts

```diff
@@ -148,37 +133,29 @@ const updateProjectUsage =
 export const fetchReadsets =
     (datasetId: TritonDataset["id"]) =>
     async (dispatch: AppDispatch, getState: () => RootState) => {
-        try {
-            const readsets = await apiTriton.listReadsetsForDataset(datasetId)
-            // console.debug(`Loaded readsets succesfully: ${JSON.stringify(readsets)}`)
-            dispatch(ReadsetsStateActions.setReadsets(readsets))
-
-            const projectId =
-                getState().datasetsState.datasetsById[datasetId]
-                    ?.external_project_id
-            if (projectId) {
-                dispatch(updateProjectUsage(projectId))
-            }
-
-            return readsets
-        } catch (err: any) {
-            throw err
+        const readsets = await apiTriton.listReadsetsForDataset(datasetId)
```

Comment:
maybe removing the try catch 'fixes' the issue but its not the right approach because we will still want to know what went wrong